contrib/get-github-actions-status.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/env python3
+
+import json
+import sys
+import urllib.parse
+import urllib.request
+from time import sleep
+
+
+def get_all_runs():
+    url = "https://api.github.com/repos/syslog-ng/syslog-ng/actions/runs?branch=master"
+    request = urllib.request.Request(url, None, {})
+    with urllib.request.urlopen(request) as response:
+        binary_response = response.read()
+
+    return json.loads(binary_response.decode())["workflow_runs"]
+
+
+def main():
+    workflow = sys.argv[1]
+    while True:
+        runs = filter(lambda run: run["name"] == workflow, get_all_runs())
```

Comment:
You can save the usage of `filter()` if you query the specific github actions:
https://docs.github.com/en/rest/reference/actions#list-workflow-runs
e.g. `https://api.github.com/repos/syslog-ng/syslog-ng/actions/workflows/devshell.yml/runs`

Reply:
Thanks, this is much better! :)